news/_posts/2024-12-22-playtest-20241222.md

```diff
@@ -0,0 +1,78 @@
+---
+kind: article
+title: "Playtest 20241222"
+author: "PunkPun"
+created_at: 2024-12-22 22:00 +0000
+disqus_id: "playtest-20241116"
+permalink: "/news/playtest-20241222/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20241222]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/INSERTHERE):
+
+Notable changes include:
+* saboteurs can now cloak while moving
+* added delete area button to the map editor
+* improved Battlefield News dropdown
+* improved formatting of unit descriptions and encyclopedias
+* fixed Dune 2K not detecting installed music and videos
```

Comment:
Both "d2k"/"D2k" and "Dune 2000" are widely used, but this version is very rare. Can we go with "Dune 2000" here?